pax-compiler/src/parsing.rs

```diff
@@ -141,12 +141,12 @@ fn recurse_pratt_parse_to_string<'a>(
                 let op0_out = match op0.as_rule() {
                     Rule::xo_literal => {
                         //return the literal exactly as it is
-                        format!("({}_isize)", op0.as_str())
+                        format!("({}isize)", op0.as_str())
```

Comment:
wont this break?

Reply:
Think it's fine! "34usize" "4.0f32", "40i128" are all legal rust literals